system-test/cluster.ts

```diff
@@ -67,6 +67,7 @@ describe('Cluster', () => {
         time_created: Date.now(),
       },
     });
```

Comment:
Is it also possible to add more IDs on the cluster that is being compared in the assertion? i.e. to easily show which cluster is mismatching the ID

```
  1) Cluster << would be nice to have the cluster ID here >>
       Update cluster
         Starting from autoscaling
           should change cluster to manual scaling:

      AssertionError [ERR_ASSERTION]: Expected values to be strictly equal:

3 !== 5

      + expected - actual

      -3
      +5

      at checkMetadata (build/system-test/cluster.js:31:16)
          -> /workspace/system-test/cluster.ts:42:12
      at async Context.<anonymous> (build/system-test/cluster.js:273:17)
          -> /workspace/system-test/cluster.ts:306:9
```

Reply:
Hmm. I'm not sure how to make a dynamic title in the mocha framework, but we can definitely print out the cluster id.

Reply:
No worries, yea, whatever is easier to do, as I'm not too familiar w/ Typescript and its frameworks. So long as we see the cluster ID logged earlier in the log that corresponds w/ the test failure, we should be alright